code/style.css

```diff
@@ -1,4 +1,293 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  background: rgba(240, 232, 232, 0.918);
+  color: black;
+  margin: 0;
+  position: relative;
 }
+
+/* Nav bar */
+
+header {
+  background-color: #cfccc6;
+}
+
+.nav {
+  height: 80px;
+  display: flex;
+}
+
+.nav .logo {
+  height: 100%;
+  border-radius: 15px 50px;
+}
+
+.nav .info {
+  font-family: "Permanent Marker", cursive;
+  font-size: 40px;
+  margin-left: 10px;
+  text-align: center;
+  margin-top: 10px;
+  justify-self: end;
+}
+
+.nav .links {
+  display: flex;
+  flex: 1;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav .links .logo-tablet {
+  width: 150px;
+  border-radius: 15px 50px;
+}
+
+a {
+  text-decoration: none;
+  padding: 0 15px;
+}
+
+a:hover {
+  background-color: #6b6868;
+  padding: 10px;
+}
+
+/* Content */
+
+.content {
+  margin: 50px 200px;
+}
+
+.big-news {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 35px;
+  align-content: center;
+}
+
+.big-news .img-container {
+  justify-items: center;
+  border: 2px solid red;
+  flex: 2;
+}
+
+.big-news .info {
+  margin-left: 20px;
+  border: 2px solid #2ac1eb;
+  text-align: center;
+  font-size: larger;
+  flex: 5;
+  padding: 10px;
+}
+
+.big-news .info p {
+  text-align: center;
+  font-size: larger;
+}
+
+.zoom-img {
+  width: 100%;
+  transition: transform 0.2s;
+}
+
+.zoom-img:hover {
+  transform: scale(1.5);
+  position: relative;
+  opacity: 0.5;
+}
+
+#content-img {
+  width: 100%;
+}
+
+h2 {
+  font-family: "Permanent Marker", cursive;
+  text-align: center;
+}
+
+/* Cards */
+
+.cards-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  column-gap: 30px;
+  justify-items: stretch;
+}
+
+.cards-container .card {
+  box-sizing: border-box;
+  border: 2px solid #2ac1eb;
+  font-size: larger;
+  text-align: center;
+  border-radius: 15px 50px;
+  box-shadow: 5px 10px hsl(192, 76%, 77%);
+  margin-bottom: 30px;
+  animation: pulse 5s infinite;
+}
+
+.cards-container .card:hover {
+  transform: scale(1.1);
+  position: relative;
+}
+
+.cards-container .card-img {
+  max-height: 300px;
+  height: 100%;
+  overflow: hidden;
+  border-radius: 15px 50px;
+}
+
+/* animation */
+
+@keyframes pulse {
+  0% {
+    background-color: #bddbda;
+  }
+  100% {
+    background-color: #5cbe88;
+  }
+}
```

Comment:
W O O O W!! This is so nice to see 💥 
Please continue to explore the animation with code, if you haven't already then check out: 
https://animista.net/